pkg/service/repair/service_repair_integration_test.go

```diff
@@ -483,6 +529,109 @@ func TestServiceGetTargetIntegration(t *testing.T) {
 	}
 }
 
+func TestServiceRepairOneJobPerHostIntegration(t *testing.T) {
+	session := CreateScyllaManagerDBSession(t)
+	h := newRepairTestHelper(t, session, repair.DefaultConfig())
+	clusterSession := CreateSessionAndDropAllKeyspaces(context.Background(), t, h.Client(), ManagedClusterHosts())
+
+	const (
+		ks1           = "test_repair_rf_1"
+		ks2           = "test_repair_rf_2"
+		ks3           = "test_repair_rf_3"
+		t1            = "test_table_1"
+		t2            = "test_table_2"
+		maxJobsOnHost = 1
+	)
+	createKeyspaceWithRF(t, clusterSession, ks1, 1, 1)
+	createKeyspaceWithRF(t, clusterSession, ks2, 2, 2)
+	createKeyspaceWithRF(t, clusterSession, ks3, 3, 3)
+	WriteData(t, clusterSession, ks1, 5, t1, t2)
+	WriteData(t, clusterSession, ks2, 5, t1, t2)
+	WriteData(t, clusterSession, ks3, 5, t1, t2)
+
+	t.Run("repair schedules only one job per host at any given time", func(t *testing.T) {
+		ctx := context.Background()
+
+		target := h.generateTarget(map[string]any{
+			"fail_fast":               true,
+			"single_host_parallelism": 1,
```

Comment:
It would be nicer to use `maxJobsOnHost` here instead of straight 1 (possible var rename).

Reply:
makes sense, fixed